src/common/chat/command_handlers/character_command_handlers.ts

```diff
@@ -148,7 +159,11 @@ async function handleCheckCommand(
   }
 
   const rolls = rollDice(20, parsed.advantage === 0 ? 1 : 2);
-  const modifier = getIgnoringCase(character.checkBonuses, skill[0]);
+  const modifier = getIgnoringCase(
```

Comment:
Since we're merging the ability bonuses in the CharacterSheetData, this additional step shouldn't be required, right?

I think it would just work if we changed:

const skillCompleter = Autocompleter.create(SKILL_ORDER):

const skillCompleter = Autocompleter.create(SKILL_ORDER.concat(ABILITY_ORDER)):

Reply:
Good call with the concat, and we don't need the boolean check. I do need to merge the bonuses again here though so I've updated with that

Reply:
You're right, the test was failing but that's just because they weren't merged in the test. I've swapped it back and updated the test